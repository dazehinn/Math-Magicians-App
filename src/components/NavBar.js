import { NavLink } from 'react-router-dom';
import styles from './Navbar.module.css';

const Navbar = () => (
  <nav className={styles.navbar}>
    <h1 className={styles.textLog}>Math Magicians</h1>
    <ul>
      <li>
        <NavLink to="/">Home</NavLink>
      </li>
      <li>
        <NavLink to="calculator">Calculator</NavLink>
      </li>
      <li>
        <NavLink to="quote">Quote</NavLink>
      </li>
    </ul>
  </nav>
);

export default Navbar;
