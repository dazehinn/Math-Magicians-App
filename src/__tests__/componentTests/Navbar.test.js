// import Calculator from '../../components/calculator';
import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import NavBar from '../../components/NavBar';

const MockNavBar = () => (
  <BrowserRouter>
    <NavBar />
  </BrowserRouter>
);

describe('NavBar Component', () => {
  test('Tree', () => {
    const tree = renderer.create(<MockNavBar />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
