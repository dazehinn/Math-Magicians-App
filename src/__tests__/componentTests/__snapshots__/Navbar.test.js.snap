// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar Component Tree 1`] = `
<nav
  className="navbar"
>
  <h1
    className="textLog"
  >
    Math Magicians
  </h1>
  <ul>
    <li>
      <a
        aria-current="page"
        className="active"
        href="/"
        onClick={[Function]}
      >
        Home
      </a>
    </li>
    <li>
      <a
        className=""
        href="/calculator"
        onClick={[Function]}
      >
        Calculator
      </a>
    </li>
    <li>
      <a
        className=""
        href="/quote"
        onClick={[Function]}
      >
        Quote
      </a>
    </li>
  </ul>
</nav>
`;
